config/example-backup.sh

```diff
@@ -71,4 +71,4 @@ GUI_OVERWRITE_ICON="$PWD/icon.png" # custom icon for notifications (needs absolu
 # export SQL_BACKUP_DESTINATION="/root/backup/db"
 
 # backup specified databases, or all if you do not pass any parameter
-# ../tools/databasedump.sh "db1 db2 db3"
+# "$( dirname "$0" )/tools/databasedump.sh "db1 db2 db3""
```

Comment:
I think you really messed up the quotes here.

![image](https://user-images.githubusercontent.com/11966684/57031921-4d7d9900-6c49-11e9-98b0-80a0d63de4bf.png)

```suggestion
# "$( dirname "$0" )/tools/databasedump.sh" "db1 db2 db3"
```